### DISCUSSION TITLE
Fief cloud is not working - is there some status page?

### DISCUSSION THREAD

**Discussion by filipagh** [fief-dev/fief#222] [General]
Hello i cant log in to admin or my app - request to get token for code result in 500

![image](https://github.com/fief-dev/fief/assets/22474494/3c588ee3-6911-467b-be9a-b22bdcbdc75f)

**Comment by frankie567** [comment]
Is it still happening? Server seems healthy here, maybe it had a small hiccup.

We don't have a status page for now, but this would indeed be useful!

**Reply by filipagh** [reply]
now its working, thanks
